airlock/workspace_api.py

```diff
@@ -0,0 +1,53 @@
+import dataclasses
+import pathlib
+
+from django.conf import settings
+from django.urls import reverse
+
+
+@dataclasses.dataclass(frozen=True)
+class PathItem:
```

Comment:
It'd be nice to have a docstring explaining what this is for and why it's worth wrapping `pathlib.Path`.